### PR TITLE
Add limited support API

### PR DESCRIPTION
**Pull request by zgalor**

### CODE REVIEW

**Comment by zgalor** [comment]
@tzvatot PTAL

File: model/clusters_mgmt/v1/limited_support_reason_type.model [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+	// Summary of the reason.
+	Summary String
+
+	// URL with a link to a detailed description of the reason.
+	Details String
+
+	// Optional link to a template with summary and details.
+	link Template LimitedSupportReasonTemplate
+
+    // The time the reason was detected.
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/limited_support_reason_type.model:33]: indentation is off

File: model/clusters_mgmt/v1/limited_support_reason_type.model [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	CreationTimestamp Date
+
+	// Indicates if the reason was detected automatically or manually.
+    // When creating a new reason this field should be empty or "manual".
```
Comment (tzvatot) [inline model/clusters_mgmt/v1/limited_support_reason_type.model:37]: indentation is off

**Comment by tzvatot** [comment]
Aside the NP, LGTM

**Comment by tzvatot** [comment]
LGTM

**Pull request merged**